mobile_app/test/test_trigger_build.py

```diff
@@ -0,0 +1,139 @@
+"""
+Tests for trigger app build script
+"""
+
+import git
+import json
+from mock import Mock, patch
+import os
+import tempfile
+from unittest import TestCase
+
+from mobile_app import trigger_build
+
+TEST_BRANCH = "test-branch"
+
+
+class Aborted(Exception):
+    """
+    Helper exception for indicating system exit instead of
+    actually exiting the process
+    """
+    pass
+
+
+def _trigger_build(repo_path, environ):
```

Comment:
Curious: Is there a reason that these helper functions are not declared as internal helper methods within the Test class?  If they remain as functions, consider moving them to the end of the file.